### ISSUE TITLE
Vulnerabilities in eta

### ISSUE THREAD

**Issue by makkes** [weaveworks/weave-gitops#3354]
Snyk is reporting vulnerabilities for one of docusaurus' dependencies:

```
  1/7: Cross-site Scripting (XSS) [Medium Severity]
    Via: @docusaurus/core@2.3.0 => eta@1.12.3
    Fixed in: eta 2.0.0
[...]
```

[Example run that failed](https://github.com/weaveworks/weave-gitops/actions/runs/4069662935/jobs/7009648596).

This currently leads to CI check failures on any PRs.

**Comment by Callisto13** [comment]
Docusaurus merged the bump yesterday https://github.com/facebook/docusaurus/pull/8610

**Comment by makkes** [comment]
Let's hope they don't stick to their past release candence to get this out.

**Comment by makkes** [comment]
Judging by the PRs merged at this very moment it looks like 2.3.1 will be released soon. Let's wait for that to happen so we can easily upgrade without having to pin transitive deps.
